apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -365,7 +365,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
   blocklyWrapper.WorkspaceSvg.prototype.resize = function() {
     googleBlocklyBlocklyResize.call(this);
     if (cdoUtils.getToolboxType() === ToolboxType.UNCATEGORIZED) {
-      // this.flyout_?.resize();
+      this.flyout_?.resize();
```

Comment:
This won't break other stuff when we merge into staging right?

Reply:
This is just a reversion to what is already in staging.  I'm not sure any more why I had to comment it out :)

Reply:
Oh cool gotcha :)